### ISSUE TITLE
failed to install dotnet4

### ISSUE THREAD

**Issue by cyberb** [scottyhardy/docker-wine#97]
dotnet4 is downloaded fine and gui installation starts but fails after a minute if installing on about 10% progress (two bars).
Not sure if it is a misconfiguration of docker-wine or not, this doc says it should be fine:
https://forum.winehq.org/viewtopic.php?t=33090

Running on Linux

**Comment by cyberb** [comment]
My bad, running the same thing on another machine worked.

**Comment by cyberb** [comment]
Strange thing, on one machine with Ubuntu 20.04 it is installing fine, but on a RedHat it is not.

First attempt always fails very quickly, second and subsequent are getting a bit further and fail consistently there.
Could you try installing dotnet4 and see if it works for you?
Also is there any way to debug this?

**Comment by scottyhardy** [comment]
Hi @cyberb, thanks for reporting the issue.  Looks like an old bug has crept back in, originally resolved in issue #92.  For whatever reason, using Ubuntu 20.04 (focal) as the base causes some installs to fail.  As a workaround I've now just set the build script to use Ubuntu 19.10 (eoan) rather than try to detect the OS version to use.  I was able to install dotnet4 (eventually - it takes quite some time) using the command:

```bash
docker-wine winetricks dotnet40
```

If you're just using the `docker-wine` script, then probably a good idea to trash the current docker volume just in case there's some garbage from failed installs causing issues with `docker volume rm winehome`
